code/game/objects/items/storage/belt.dm

```diff
@@ -241,6 +241,24 @@
 		/obj/item/extrapolator
 		))
 
+/obj/item/storage/belt/medical/ert
+	name = "emergency response medical belt"
+	desc = "A belt containing field surgical supplies for use by medical response teams."
+
+/obj/item/storage/belt/medical/ComponentInitialize()
```

Comment:
Shouldn't this be `/obj/item/storage/belt/medical/ert/ComponentInitialize()` and not `/obj/item/storage/belt/medical/ComponentInitialize()`?

Reply:
Good catch